ocsptools/ocsp_tools.go

```diff
@@ -29,6 +29,23 @@ func PrintCert(cert *x509.Certificate) error {
 	return nil
 }
 
+// ParseCertificateFile takes a path to a certificate and returns a parsed certificate
+func ParseCertificateFile(certFile string) (*x509.Certificate, error) {
+	cert, err := ioutil.ReadFile(certFile)
```

Comment:
did you find out the canonical way to close file when err?

Reply:
Looking at the implementation of ioutil.ReadFile https://golang.org/src/io/ioutil/ioutil.go?s=1503:1549#L42, it doesn't look like I need to worry if there is an err. If it errs on os.Open, then the file was never opened in the first place, otherwise if it errs, since defer f.Close is right after os.Open, the file is guaranteed to close if ioutil.ReadFile returns.